content/applications/general/voip/axivox/conference_calls.rst

```diff
@@ -86,7 +86,7 @@ is then prompted to enter the :guilabel:`Access code` to enter the conference.
 Start call in Odoo
 ==================
 
-Anywhere in the Odoo database, open the *VoIP* widget, by clicking the :guilabel:`📞 (phone)` icon,
+Anywhere in the Odoo database, open the *VoIP* widget, by clicking the :guilabel:`☎️ (phone)` icon,
 located in the upper-right corner. Then, dial the specific extension number for the conference, and
 click the :guilabel:`📞 (phone)` icon.
```

Comment:
Missed an emoji I think. Added some more context too so it didn't read as redundant.
```suggestion
Anywhere in the Odoo database, open the *VoIP* widget, by clicking the :guilabel:`☎️ (phone)` icon,
located in the upper-right corner. Then, dial the specific extension number for the conference, and
click the :guilabel:`☎️ (phone)` icon again, this time on the *VoIP* widget itself.
```

Reply:
Whoop. I corrected myself— what you had originally was accurate. Disregard please!